itests-runner/resources/prepare-env.sh

```diff
@@ -3,8 +3,8 @@
 set -e
 
 function install_package {
-    rpm -qa | grep $1 > /dev/null
-    if [ "$?" -eq "1" ]; then
+    rpm -q $1 > /dev/null
+    if [ ! $? -eq 0 ]; then
```

Comment:
i think this is just a weird way to write `&&`; maybe we should be able to do `rpm .. && { echo "installing"; sudo yum install}`?

but there's probably no easy way to quickly test it, and this just used to work, so it should still work. So, meh.